docs/user-guides/client_config_guide.md

```diff
@@ -1,12 +1,138 @@
+
+| Option: |	Description: | Usecases: |
+| --- | --- | --- |
+| Mixed echo cancellation | This is the basic Option: It will process all loudspeaker outputs bundled together.
+This is less accurate than the Multichannel option, but will also use less CPU. 	| Sufficient for setups with loudspeakers near to the microphone. |
+| Multichannel echo cancellation | Extended option: This will process all audio channels seperately, this is more accurate, but will result in higher CPU usage. |	For setups with (multiple) loudspeakers farther away from the microphone. |
+| Disabled | Disables echo cancellation (not recommended).  | You use a very good headset. Or you use external echo cancellation (for example via pulseaudio on Linux). Or echo cancellation is causing problems. |
+
+#### Transmission
+
+You can choose between three transmission modes:
+
+1. **Push to Talk**
+
+Your voice is only transmitted when you press a button.
+You can set the button in the `Shortcuts`-Menu (see below).
+
+![Here should be a picture](pictures/audio/push-to-talk.png)
+
+
+2. **Voice Activity**
+
+Your voice is only transmitted when a certain limit (loudness) is reached.
+All noise below the limit will be canceled.
+
+*Tip:* Use the `Audio Wizard`(`Configure`->`Audio Wizard`) to configure this option correctly.
+
+<!-- what is the difference between `Amplitude` and `Signal to noise` ? -->
+
+![Here should be a picture](pictures/audio/voice-activity.png)
+
+The coloured bar will show the noise-level when you speak, adjust `Silence below` (everything below the marker will not be transmitted) and `Speech above` (everything above the marker will be transmitted) accordingly.
+
+3. **Continuous**
+
+All Audio from your microphone will be transmitted continuously.
+
+**Note:** You should usually **not** use this option, as it can be very annoying for listeners.
+It is intended for special usecases (Music bots, Recording etc.) or when you use external tools for audio filtering instead.
+
+#### Compression
+
+**Quality:**
+Sets the maximum bandwidth the client will try to accomplish (it is limited by the Server you are connected to).
+Setting it higher will increase the audio quality.
+Only set it higher if your internet connection is good enough.
+<!-- maximum? -->
+
+<!-- Audio per packet: What does it do? Which is option is better for which scenario? -->
+
+#### Audio Processing 
+
+**RNNoise**
+
+RNNoise is a noise suppression library.
+It is intended to filter background noises from the audio that you send sent to other users.
+RNNoise uses a deep learning algorithm so it should work better than most regular filters.
+
+More information can be found on various websites:
+
+- [RNNoise GitHub repo](https://github.com/xiph/rnnoise)
+- [Article at Mozilla.org on RNNoise](https://hacks.mozilla.org/2017/09/rnnoise-deep-learning-noise-suppression/)
+
+
+
+### Shortcuts
+
+![Here should be a picture](pictures/shortcuts.png)
+
+<!-- insert gif? -->
```

Comment:
Given that our shortcut menu currently is not the most intuitive thing to use, this might be a good idea :+1:

Reply:
Will see how good or bad that will work :D.

Reply:
I did some testing with the gifs and right now it seems I am not able to make that work properly (filesizes to big, picture to small, quality to bad). Quess I do something wrong.

But I will keep trying; as an alternative I might just create a very short gif with the main steps (clicks) manually...